src/app/zap-templates/zcl/temperature-measurement-cluster.xml

```diff
@@ -25,8 +25,8 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <attribute side="server" code="0x0000" define="TEMP_MEASURED_VALUE" type="INT16S" writable="false" optional="false">measured value</attribute>
-    <attribute side="server" code="0x0001" define="TEMP_MIN_MEASURED_VALUE" type="INT16S" min="-38221" max="65535" writable="false" optional="false">min measured value</attribute>
-    <attribute side="server" code="0x0002" define="TEMP_MAX_MEASURED_VALUE" type="INT16S" min="-38221" max="65535" writable="false" optional="false">max measured value</attribute>
+    <attribute side="server" code="0x0001" define="TEMP_MIN_MEASURED_VALUE" type="INT16S" min="-38221" max="65535" writable="false" default="0x8000" optional="false">min measured value</attribute>
```

Comment:
What section of the spedc are these min/max/default values defined? I tried to find them but somehow failed.

When looking at this, I would expect that min to be (INT16_max) and max to be (int16_min) so that they are obvious defaults and change. Having the defaults for both min an  max seems odd to me (extra odd having them specified in hex when the the min/max values are in integer):
   - 65535 does not look like a valid value for a signed int16 - looks like a off-by-one unsigned int16
   - "-38221" looks very odd as well (-32768 seems  the right value for signed int16_t
   - having both min/max equal and hex seems odd (the hex value, if treated as positive, seems just out of range of signed int16_t)

Would really like to understand the spec here.

Reply:
Read https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/app_clusters/TemperatureMeasurement.adoc

-27315 is the correct min value.  Please update that as well.

Reply:
Max also needs to be updated 32766 and 32767. Or better yet, if supported, use hex for  those.

Reply:
Yeah, this got pretty broken in #7224.  The right min is -27313 (i.e. `0x954E - 65535` as a signed number) and the right max is `32766` for MinMeasuredValue, `32767` for MaxMeasuredValue.

Reply:
You are right. I have fixed that now.